src/overlays/actors/ovl_Fuusen/ac_fuusen.c

```diff
@@ -41,10 +41,9 @@ ActorProfile Fuusen_Profile = {
 };
 
 void aFSN_actor_ct(Actor* thisx, Game_Play* game_play) {
-    static xyz_t Init_Size = { 0.01f, 0.01f, 0.01f };
     Fuusen* this = THIS;
     SkeletonInfoR* skeletonInfo = &this->skeletonInfo;
-    xyz_t size = Init_Size;
+    xyz_t Init_Size = { 0.01f, 0.01f, 0.01f };
```

Comment:
I'm confused by this but I think it should keep the static keyword?

Reply:
it wont match if it does

Reply:
I thought it made more sense for them to write it like this than what I previously put, but only realised when I had the same issue here in bee